### DISCUSSION TITLE
Why sha512 vs xxh128 in CacheStorage

### DISCUSSION THREAD

**Discussion by ruudk** [web-auth/webauthn-framework#663] [Q&A]
Why is sha512 used for storing, but xxh128 to get it? How will this ever work? I might be missing something here. 

https://github.com/web-auth/webauthn-framework/blob/9b9e5daeb12a7d31a61ac2e2e24459f6a1280440/src/symfony/src/Security/Storage/CacheStorage.php#L19-L46

**Comment by Spomky** [comment]
Hi @ruudk,

This is a bug... Should be xxh128 everywhere
